open_alchemy/build/__init__.py

```diff
@@ -218,14 +220,15 @@ def dump(
         raise exceptions.BuildError(str(exc)) from exc
 
 
-def execute(*, spec: typing.Any, name: str, path: str) -> None:
+def execute(*, spec: typing.Any, name: str, path: str, format_: str = None) -> None:
```

Comment:
Does `format` this need to be typing.Optional[str]? Also, given that it defaults to `None` on the interface, does it need to default to `None` here again?